pages/index.tsx

```diff
@@ -1,30 +1,45 @@
 import Section from "@/components/Section"
+import Image from "next/image"
+import latestSermon from '../images/latest-sermon.png'
+import welcomeToTMI from '../images/welcome-to-tmi.png'
+import facebookLogo from '../images/facebook.png'
+import instagramLogo from '../images/instagram.png'
+import youtubeLogo from '../images/youtube.png'
 
 export default function Home() {
   return (
     <div>
       <Section>
-        <div className="pt-72">
-          <p className="text-3xl font-thin">LATEST SERMON</p>
-          <p className="font-bold text-7xl mb-8 mt-5">The Premise of the Promise <br /> of a Happy New Year</p>
-          <button className="bg-tmi-yellow py-1.5 px-7 font-bold rounded-3xl mr-3">Watch Now</button>
-          <button className="bg-transparent border border-white rounded-3xl py-1.5 px-7">See More Sermons</button>
+        <div className="relative bg-gradient-to-t from-tmi-gradient-blue to-transparent w-full h-full">
+          <Image src={latestSermon} fill alt="Latest Sermon" className="w-full h-full object-cover absolute -z-10" />
+          <div className="pt-72 mx-10">
+            <p className="text-3xl font-thin">LATEST SERMON</p>
+            <p className="font-bold text-7xl mb-8 mt-5">The Premise of the Promise <br /> of a Happy New Year</p>
```

Comment:
mao ni ang murag title jud sa page sa? better nga i `h1` nimo for good SEO practice, kay usually ang `h1` ga indicate na nga mao na ang title sa kana nga web page

Reply:
resolved